packaging/debian-sid/patches/skip-tests-seccomp-backend.patch

```diff
@@ -0,0 +1,31 @@
+Index: snapd.mvo/interfaces/seccomp/backend_test.go
+===================================================================
+--- snapd.mvo.orig/interfaces/seccomp/backend_test.go
++++ snapd.mvo/interfaces/seccomp/backend_test.go
+@@ -355,6 +355,8 @@ func (s *backendSuite) TestSocketcallIsA
+ }
+ 
+ func (s *backendSuite) TestSocketcallIsNotAddedWhenNotRequired(c *C) {
++	c.Skip("skipped until debian has check.v1 from 2018. Not() is broken in the old version")
```

Comment:
I think we could just tweak the test to avoid using Not()

Reply:
This patch is gone in my downstream package. The corresponding library was fixed.